training.rst

```diff
@@ -0,0 +1,113 @@
+resources as well as hints for trainers about how to present OMERO during a training session.
+
+Resources
+---------
+
+- Community contributions and setups: `Github issue <https://github.com/ome/omero-guides/issues/107>`_.
+
+- `Useful scripts <https://github.com/ome/training-scripts>`_ for setting up the training server.
+
+
+Setup of scripts and OMERO.cli environment
+------------------------------------------
+
+- This guide assumes that you have installed an OMERO.server for training purposes.
+
+- Clone the `training-scripts <https://github.com/ome/training-scripts>`_ repository::
+
+    $ git clone https://github.com/ome/training-scripts.git
+
+- Set up a OMERO.cli as specified under `CLI installation <https://docs.openmicroscopy.org/omero/latest/users/cli/installation.html>`_. Typically, this environment will be used on your local machine. Alternatively, you can use the OMERO.cli environment of the OMERO.server.
+
+Setup of groups and users
+-------------------------
+
+Prepare groups and users as listed in the `provided template sheet <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users_setup>`_. Assuming you have the `template sheet <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users_setup>`_ in the same directory as the script `create_groups_users.sh <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users.sh>`_, you can run::
+
+    $ HOST=$YOUR_SERVER_ADDRESS PASSWORD=$PASSWORD_FOR_ROOT bash create_groups_users.sh
+
+which will create 50 users in your database.
+The users are members of four main OMERO.groups, which cover
+the group permissions allowed in OMERO. The ``Read-annotate`` group ``Lab 1`` is the main group used in the trainings,
+which contains most of the data in a typical training. The setup in the `template sheet <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users_setup>`_ sets this ``Lab 1`` group as a default group in OMERO for all users.
+
+Rename users to have first and last names of real people (the list of famous scientist names is used) running `the renaming script <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/rename_users.py>`_.
+
+Import images
+-------------
+
+`Import data in-place <https://omero-guides.readthedocs.io/en/latest/upload/docs/import-cli.html#in-place-import-using-the-cli>`_ for all users using `import bash script <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/in_place_import_as.sh>`_ and the `publicly available data provided <https://downloads.openmicroscopy.org/images/>`_ or use your own data or `data downloaded from IDR <https://idr.openmicroscopy.org/about/download.html>`_. Shell into the machine where you have installed your OMERO.server (necessary for in-place import) and, assuming that you have for example your data in a folder `my-folder` which is visible from that machine::
```

Comment:
Possibly this section could use an intro paragraph: "We like to use [in-place import] because it avoids duplicating the data on disk, but if you are only importing a small amount of data you may find non-in-place import (described below) is more convenient"?  (I guess "convenient" depends on their setup, but it seems trickier to download the data into a suitable place on the server?)

Reply:
fixed in https://github.com/ome/omero-guides/pull/236/commits/68522283f4402ea8334b450b705ed8a77c180fc1